src/test/app/forms/models/name.ts

```diff
@@ -3,34 +3,36 @@
 
 import { expect } from 'chai'
 import { Validator } from 'class-validator'
-import { expectValidationError } from './validationUtils'
-import * as randomstring from 'randomstring'
+import { expectValidationError, generateString } from './validationUtils'
 import { Name, ValidationErrors } from 'app/forms/models/name'
 
 describe('Name', () => {
 
   describe('constructor', () => {
+
     it('should set the primitive fields to undefined', () => {
-      const name = new Name()
+      const name: Name = new Name()
       expect(name.name).to.be.undefined
     })
   })
 
   describe('deserialize', () => {
+
     it('should return an instance initialised with defaults for undefined', () => {
       expect(new Name().deserialize(undefined)).to.eql(new Name())
     })
 
     it('should return an instance from given object', () => {
-      const name = 'My Full Name'
-      const result = new Name().deserialize({
+      const name: string = 'My Full Name'
```

Comment:
these typings seem pointless, I think there's an eslint rule we can activate to stop pointless typing.

Reply:
There must be one for it, angular uses it.